src/style.css

```diff
@@ -1,3 +1,82 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
+  text-transform: uppercase;
+}
+
 body {
   margin: 0;
+  box-sizing: border-box;
+}
+
+.container {
+  padding: 0 50px;
+  justify-content: space-between;
+  display: flex;
+  align-items: center;
+}
+
+.logo {
+  display: flex;
+  align-items: center;
+}
+
+/* .nav {
+  display: flex;
+  height: 60px;
+  flex-direction: row;
+  justify-content: stretch;
+} */
+
+.nav_list {
+  display: flex;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  margin-right: -20px;
+}
+
+.nav_item {
+  display: inline-block;
+  margin-right: 20px;
+}
+
+.nav_link {
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+  color: rgba(0, 0, 0, 1);
+  margin: 0;
+  height: 60px;
+  line-height: 60px;
+  position: relative;
+}
+
+.is-active {
+  text-decoration: none;
+  color: rgba(0, 172, 220, 1);
```

Comment:
Why don't we create a variables for colors? 

:root {
  --col-text: ...: ;
  --col-ative: ....;
}